iot_input/models/iot_device_input.py

```diff
@@ -63,6 +70,29 @@ def call_device(self, value):
             self._add_action_vals(value, res))
         return res
 
+    @api.model
+    def get_device_input(self, serial, passphrase, value):
+        device_input = self.with_context(
```

Comment:
Why don't you maintain the current hook `get_device`

Reply:
I want to put active_test in context to search also for inactive inputs

Reply:
Why do you need inactive inputs??? If it is inactivated, you don't want to let him access your system

Reply:
Ok